### PR TITLE
removes database creating and grants - this is automatically configur…

### PR DESCRIPTION
**Pull request by dgo19**
…ed by postgres env variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,53 +99,27 @@ After PostgreSQL started up, it's time to adjust some configuration for the quay
 
 ```
 ##### Start psql session in quay-database container
-% podman exec -it quay-database /usr/bin/psql
+% podman exec -it quay-database /usr/bin/psql -d quaydb
```
Comment (ableischwitz) [inline README.md]: Docs use `psql -d quay -U postgres` - even though this shouldn't matter, we should try to be as close as possible to the official docs.

File: README.md [hunk 2]
```diff
@@ -99,53 +99,27 @@ After PostgreSQL started up, it's time to adjust some configuration for the quay
...
-GRANT
 <ctrl+d>
 ```
 
 ## Configure Clair database
 
 ```
 ##### Start psql session in quay-database container
-% podman exec -it quay-clair-database /usr/bin/psql
+% podman exec -it quay-clair-database /usr/bin/psql -d clair
```
Comment (ableischwitz) [inline README.md]: Same here - so `podman exec -it quay-clair-database /usr/bin/psql -d clair -U postgres` should be used.

**Review by ableischwitz — COMMENTED** [review]
While I would like to keep the instructions for user creation, the docs also don't have instructions to do so.
After adjusting the `psql` statements, I will merge it.

File: README.md (hunk 1 above)
Comment (dgo19) [reply README.md]: added -U postgres

File: README.md (hunk 2 above)
Comment (dgo19) [reply README.md]: added -U postgres

**Pull request merged**